ext/TimeZonesRecipesBaseExt.jl

```diff
@@ -1,3 +1,8 @@
+module TimeZonesRecipesBaseExt
```

Comment:
Although the Pkg.jl documentation use this convention in their docs I think having "TimeZones" named in the extension is a little too verbose for my liking.

```suggestion
module RecipesBaseExt
```
Will need to also rename the file.

Reply:
It's not documented, but apparently the convention is to have the 'host' package's name first to avoid name collisions, which are problematic. 

(pointed out to be by @ararslan in an NLopt PR: https://github.com/JuliaOpt/NLopt.jl/pull/204#discussion_r1293669487)

Reply:
Thanks. I would speculate this is probably due to how `.ji` files currently work. Okay, let's leave this then.